downstream/modules/platform/ref-complex-mesh-topology.adoc

```diff
@@ -2,7 +2,7 @@
 
 [id="ref-complex-mesh-topology"]
 
-= Complex automation mesh design pattern example Three controllers, 2 exec nodes + multi hops with exec nodes
+= Complex automation mesh design example
 
 
 [role="_abstract"]
```

Comment:
Typo in the {AutomationController} attribute

Reply:
Fixed, thanks for catching that @TarikFD